### DISCUSSION TITLE
1-Click PgBouncer Installation - Connection Pooler for PostgreSQL

### DISCUSSION THREAD

**Discussion by Probabilista** [supabase/supabase#795] [Feature Requests] [answered]
Applications built on serverless architectures can have a large number of open connections to the database server, and may open and close database connections at a high rate, exhausting database memory.

PgBouncer is a lightweight connection pooler for PostgreSQL, and a one-click installation setup could be nice :)

**Comment by steve-chavez** [comment, marked as answer]
Hey @Probabilista,

Pgbouncer addition is being discussed here: https://github.com/supabase/postgres/pull/33. Feel free to chime in.

We also had a previous discussion about this: https://github.com/supabase/supabase/discussions/775

Note that the REST API already has a pool. So instead of connecting directly to pg in serverless, you could issue a request to the REST API(through supabase-js) and that will take care of managing connections.

**Reply by Probabilista** [reply]
Oops. Sorry. Didn't see issue 33.
